layouts/partials/header.html

```diff
@@ -1,92 +1,126 @@
+          </div>
+        </div>
+      </li>
+      <li>
+        <div class="dropdown">
+          <button type="button" class="dropdown__toggle" data-dropdown-toggle="learn-dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+            <span class="rvt-dropdown__toggle-text">About Rivet</span>
+            <svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+              <path fill="currentColor" d="M8,12.46a2,2,0,0,1-1.52-.7L1.24,5.65a1,1,0,1,1,1.52-1.3L8,10.46l5.24-6.11a1,1,0,0,1,1.52,1.3L9.52,11.76A2,2,0,0,1,8,12.46Z"/>
+            </svg>
+          </button>
+          <div class="rvt-dropdown__menu" id="learn-dropdown" role="menu" aria-hidden="true">
+            <a href="{{ trim .Site.BaseURL "/" }}/about/" {{ if eq $currentNav.URL "/about/" }}aria-current="page"{{ end }}>What is Rivet?</a>
+            <a href="{{ trim .Site.BaseURL "/" }}/blog/" {{ if eq $currentNav.URL "/blog/" }}aria-current="page"{{ end }}>Blog</a>
+            <a href="{{ trim .Site.BaseURL "/" }}/components/information/roadmap/" {{ if eq $currentNav.URL "/components/information/roadmap/" }}aria-current="page"{{ end }}>Roadmap</a>
+            <a href="{{ trim .Site.BaseURL "/" }}/components/information/changelog/" {{ if eq $currentNav.URL "/components/information/changelog/" }}aria-current="page"{{ end }}>Changelog</a>
+          </div>
+        </div>
+      </li>
+      <li><a href="/workshop">Community Workshop</a></li>
+    </ul>
+  </nav>
+  <div class="rvt-supplemental-menu">
+    <div class="rvt-dropdown rvt-notifications">
+      <button role="button" class="rvt-notifications__toggle" ref="notificationButton" data-dropdown-toggle="notification-list" aria-expanded="false">
+        <span class="rvt-sr-only">Show notifications</span>
+        <span aria-hidden="true">
+          <svg class="rvt-open-icon" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+            <path fill="currentColor" d="M9.41,8l5.29-5.29a1,1,0,0,0-1.41-1.41L8,6.59,2.71,1.29A1,1,0,0,0,1.29,2.71L6.59,8,1.29,13.29a1,1,0,1,0,1.41,1.41L8,9.41l5.29,5.29a1,1,0,0,0,1.41-1.41Z"/>
+          </svg>
+          <svg class="rvt-closed-icon" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+            <path fill="currentColor" d="M14.57,12.06,13,9.7V6A5,5,0,0,0,3,6V9.7L1.43,12.06a1.25,1.25,0,0,0,1,1.94H6a2,2,0,0,0,4,0h3.53a1.25,1.25,0,0,0,1-1.94ZM8,12H3.87L5,10.3V6a3,3,0,0,1,6,0v4.3L12.13,12Z"/>
+          </svg>
+        </span>
+      </button>
+      <div id="notification-list" class="rvt-notifications__menu" ref="menuElement" aria-hidden="true">
+        <ol class="rvt-notifications__menu-list">
+          {{ range first 3 (where .Pages "Section" "blog") }}
+            <li>
+              {{ if (isset .Params "external_url") }}
+                <a href="{{.Params.external_url}}">
```

Comment:
Let's use a blank space after the opening and before the closing of all mustache-like statements like you've done here on the `range` and `if (isset ...)` statements above.

```handlebars
<!-- So this... -->

{{.Params.external_url}}

<!-- Becomes this... -->

{{ .Params.external_url }}
```